test/deploy/kubernetes/deploy.sh

```diff
@@ -5,13 +5,70 @@ NAMESPACE="default"
 TIMEDELAY="5"
 INTRDELAY="2"
 DEF3PLOGS="/tmp/3rdparty_services_install.log"
+REDEFPODS=false
 
 function print_header {
     echo -e "--------------------------------------------------------------------------"
     echo -e "$1"
     echo -e "--------------------------------------------------------------------------"
 }
 
+function show_usage {
```

Comment:
There should be a similar file called upgrade.sh containing upgrade workflow.

Reply:
Its there in the same directory.